cmd/aries-js-worker/main.go

```diff
@@ -38,13 +44,41 @@ type result struct {
 
 // All supported command handlers.
 func handlers() map[string]map[string]func(*command) *result {
-	return map[string]map[string]func(*command) *result{
-		"test": {
-			"echo":       echo,
-			"throwError": throwError,
-			"timeout":    timeout,
-		},
+	if isTest {
```

Comment:
I meant leave them in there for production :). Or, more accurately, for use at development time when I want to test integration between the output JS bundle and the wasm.

Reply:
done